manager/impl/src/main/java/org/jboss/forge/furnace/manager/impl/AddonManagerImpl.java

```diff
@@ -217,12 +217,15 @@ private AddonActionRequest createRequest(final AddonInfo requestedAddonInfo, fin
     */
    private void collectRequiredAddons(AddonInfo addonInfo, List<AddonInfo> addons)
```

Comment:
This should only look in the one repository in which the Addon is being installed.